http-gateway/service/cancelResourceCommands.go

```diff
@@ -0,0 +1,48 @@
+package service
+
+import (
```

Comment:
Hm, Wouldn't it be better to change HTTP and gRPC gateway API a bit - don't include pending metadata updates into get pending commands? Now it's really inconsistent and we can soon expect a convenient feature to delete it using one endpoint - and that's what we don't want to do. What do you think about new endpoint, only for pendingMetadataUpdates and not sending them in getPendingCommands?